__tests__/src/services/DataService.test.ts

```diff
@@ -131,26 +133,32 @@ describe('DataService', () => {
       const expectedQueryResults = expectedTickets.map((ticket) => ({
         ...ticket,
         clients: { first_name: mockClient.first_name },
-        pets: { name: mockAnimal.name },
-        team_members: { first_name: mockTeamMember1.first_name },
+        pets: { name: mockPet.name },
+        team_members: { first_name: mockTeamMember1.first_name, email: mockTeamMember1.email },
         service_category: ticket.id,
         label: 'mock app constant label',
-      }))
+      }));
       mockSupabaseClient.setTestData(expectedQueryResults);
       const expectedServiceRequestSummary = expectedQueryResults.map((data) => {
         const {
           clients, pets, team_members, id, label, ...ticket
         } = data;
         return {
-          id: id,
+          id,
```

Comment:
nit - is this change needed?

Reply:
I think linting fixed this, but in general this is good style.